bin/lamassu-update-wallet-nodes

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+const _ = require('lodash/fp')
+
+const { utils: coinUtils } = require('lamassu-coins')
+const cryptos = coinUtils.cryptoCurrencies()
+
+const common = require('../lib/blockchain/common')
+
+const PLUGINS = {
+  BTC: require('../lib/blockchain/bitcoin.js'),
+  LTC: require('../lib/blockchain/litecoin.js'),
+  ETH: require('../lib/blockchain/ethereum.js'),
+  DASH: require('../lib/blockchain/dash.js'),
+  ZEC: require('../lib/blockchain/zcash.js'),
+  BCH: require('../lib/blockchain/bitcoincash.js')
+}
```

Comment:
While this works for current coins, the script will need manual maintenance for every new coin added to the system. Is something like this possible?

```suggestion
const PLUGINS = {}
_.forEach(it => PLUGINS[it.cryptoCode] = require(`../lib/blockchain/${it.code}.js`), cryptos)
```

Reply:
I guess that eventually manual maintenance will be needed, as the coin script will need to be developed, so you can ignore this I think

Reply:
Yeah, I think we can't run from it other than placing this in a common file